kong/acl_service_test.go

```diff
@@ -137,6 +137,50 @@ func TestACLGroupGet(T *testing.T) {
 	require.NoError(client.Consumers.Delete(defaultCtx, consumer.ID))
 }
 
+func TestACLGroupGetByID(T *testing.T) {
```

Comment:
Since you are testing multiple use-cases in one test function, I would suggest to make sub-tests via `t.Run()`
It is better for code readability and also tells us which test is failing precisely.
Can refer this for the same: https://github.com/Kong/go-kong/blob/main/kong/partial_service_test.go